docs/spec/light/load_balancer.md

```diff
@@ -0,0 +1,203 @@
+# Load Balancing Module
+
+The LCD will be an important bridge between service providers and cosmos blockchain network. Suppose
```

Comment:
Not sure about this section. I don't think we want to build a load balancer in to the LCD. 

More likely we need to work on streaming events out of Tendermint and into something like Postgres and load-balancing the access to postgres.

Reply:
I think it could be useful that a single light-client distributes its requests over many full-nodes. That's what this section is mostly describing. I am not sure whether we want Gaia Light to include postgres (or some other database) as a dependency.